src/Resolver/ValueResolver/ValueResolver.php

```diff
@@ -10,18 +10,50 @@
 use PhpParser\Node\Expr\Cast;
 use PhpParser\Node\Expr\ConstFetch;
 use PhpParser\Node\Expr\FuncCall;
+use PhpParser\Node\Expr\Variable;
 use PhpParser\Node\Name;
 use PhpParser\Node\Scalar\MagicConst\Dir;
 use PhpParser\Node\Scalar\MagicConst\File;
 
 final class ValueResolver
 {
+    private bool $resolveAllPossiblePaths;
+
+    public function __construct(bool $resolveAllPossiblePaths = false)
+    {
+        $this->resolveAllPossiblePaths = $resolveAllPossiblePaths;
+    }
+
+    /**
+     * @return mixed
+     */
+    public function resolveName(Expr $expr, ?string $actualFile = null)
+    {
+        return $this->resolve($expr, $actualFile, false);
+    }
+
     /**
      * @return mixed
      */
-    public function resolve(Expr $expr, ?string $actualFile = null)
+    public function resolvePath(Expr $expr, ?string $actualFile = null)
```

Comment:
I think there should be some PathResolver which will use ValueResolver

Reply:
I was thinking abut multiple methods in ValueResolver for different types of values

Reply:
Problem is that Value resolver would need to mae some internal parameters public to allow change of behaviour as required by PathResolver - I think one value resolver with multiple methods is cleaner

Reply:
ValueResolver just resolves the value of Expression, PathResolver knows that Expression should be path and act with it as with path. For me it is clear to do not mix these two approaches

Reply:
Problem is that I would need to pass $allowPathWildcards into Value resolver somehow and I do not like to have it in public interface of ValueResolver

Reply:
Other option is to have duplicated code in two resolvers. Form these three options 1) dumplicated code 2) ugly internal parameter in value resolver 3) two methods in Valueresolver i chosed number 3

Reply:
Which would you preffer: Maybe option 2 with some better behaviour? Like parameter $unknownValuePlaceholder and PathResolver would pass *?

Reply:
Changed